<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AScanLayout</class>
 <widget class="QWidget" name="AScanLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>787</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_A1" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget_A2" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget_A3" native="true"/>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget_A4" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_A5" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_A6" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QWidget" name="widget_A7" native="true"/>
   </item>
   <item row="3" column="1">
    <widget class="QWidget" name="widget_A8" native="true"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
